ert_shared/plugins/hook_specifications/visualization.py

```diff
@@ -1,14 +0,0 @@
-from ert_shared.plugins.plugin_manager import hook_specification
-
-
-@hook_specification
-def register_visualization_plugin(handler):
```

Comment:
If you removed this I think you need to remove also 
```
.. autofunction:: ert_shared.plugins.hook_specifications.visualization.register_visualization_plugin

.. autoclass:: ert_shared.plugins.VisualizationPluginHandler
    :members: add_plugin
    :undoc-members:
```
from `docs/rst/manual/experimental/plugin_system.rst`